docs/index.html

```diff
@@ -44,6 +44,18 @@ <h2 id="swagger--summary-no-tags">Summary</h2>
         
                         </td>
                     </tr>
```

Comment:
@vickywane if you look closer in this document , it still contains obsolete tags, such as:

```
<a href="#path--notification">/notification</a>
```

Please do a thorough review of all source and clean it up before requesting another PR review.

Reply:
I don't fully understand. 

There is still a  `POST /notification` function route that is used in posting a notification from the Edge device to Cloud API.

Previously, there was a `GET` operation in the `/notification` route to get product details but that has been moved into the `GET /product` function.

Reply:
OK, that makes sense.